src/Management/ResourceBuilder.php

```diff
@@ -40,7 +40,7 @@ public function __construct()
     /**
      * @param array $data
      *
-     * @return Space|Asset|Upload|ContentType|Entry|EntrySnapshot|Locale|Webhook|WebhookCall|WebhookCallDetails|WebhookHealth|ResourceArray|PublishedContentType|User
+     * @return Space|Asset|Upload|ContentType|Entry|EntrySnapshot|Locale|Webhook|WebhookCall|WebhookCallDetails|WebhookHealth|ResourceArray|PublishedContentType|User|Organization
```

Comment:
Yes, `ResourceArray|ResourceInterface` is probably the most reasonable choice over here...